.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+---
+name: CI
+on: [push]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        buildtype: [Release, Debug]
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install Debian Packages
+        run: |
+          sudo apt update
+          sudo apt install -y cmake ninja-build
+      - name: Toolchain Cache
+        uses: actions/cache@v2
+        id: cache
+        with:
+          path: ~/toolchain
+          key: tc-${{ runner.os }}-${{ hashFiles('.github/toolchain.sh') }}
+      - name: Install Toolchain
+        if: steps.cache.outputs.cache-hit != 'true'
```

Comment:
This is not great, because if a cached toolchain is found, a newer one is never downloaded, even when a new build of the toolchain is available. I need to change this step to be unconditional, and use `curl --time-cond` like we did with Travis.

Reply:
splited off in https://github.com/Igalia/cog/issues/272

Reply:
I have updated the PR to fix this issue; what was split in #272 is using the toolchains with the stable packages :wink: